awscli/customizations/wizard/app.py

```diff
@@ -52,6 +52,8 @@ class InvalidChoiceException(Exception):
 
 
 class WizardTraverser:
+    DONE = '__DONE__'
```

Comment:
I'm wondering if it'd be better to make `Done` a real step in definition and let customers set their own dialog and button titles, also if we decide to add additional functionality to this dialog (eg. third button) customer will be able to switch it on/off

Reply:
I could see it making sense in the future. I'm just not sure how we would want to approach it in the definition. Mainly it is different than a normal prompt where it saves a value and seems like it would need something more similar to the execute step logic where it defines actions/side effects to do. I think I'd prefer deferring coming up with an interface for this until we have more examples on how we want to customize it. Or were you thinking just adding like an empty `DONE` section in the definition as a sentinel for now? An example of what you had in mind would be helpful.

Reply:
Talked offline. Going to add a reservered `__DONE__` section where all wizards must define:
```
plan:
  first_section:
    shortname: First
    description: First section stuff
    values: ...
  ...  # Do more sections
  __DONE__:
```
Defining this will give you some default dialog but if you wanted to customize it you could. So for example:
```
__DONE__:
  description: Run wizard?  # This is the default dialog title
  options: # This specifies which buttons to include `yes` and `button` are by default
    - yes
    - back
```
Then you could override the built-in options (here it will override the button text):
```
options:
  - name: yes
    description: Yep!
  - name: back
    description: Go back
```